src/main/java/com/digirati/taxonomy/manager/lookup/persistence/Dao.java

```diff
@@ -0,0 +1,14 @@
+package com.digirati.taxonomy.manager.lookup.persistence;
+
+import java.util.Optional;
+
+interface Dao<T> {
```

Comment:
In the next change coming I've got something that stitches it all together and at the time of writing this I figured it would make sense for the DAOs for each of the models to share the same contract. Now that I've got something in place for that locally, and seeing how similar the two implementations of this are, I'm not sure I care about the shared contract, but I think there's a good case for making them extend the same abstract class with shared implementation

Reply:
Is the abstract class suggestion so you can re-use common "base" functionality from the abstract class?

Reply:
The only differences between the two impls is how they build the relevant object from the ResultSet and the Strings "concept" vs "concept scheme" in the logs